### PR TITLE
Builder style

### PR DESCRIPTION
**Pull request by rrsslnd**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -3,5 +3,6 @@ import FyllUtRouter from "../src/Forms/FyllUtRouter";
 import Template from "../src/template";
 import { AppConfigProvider } from "../src/configContext";
 import AmplitudeProvider from "../src/context/amplitude";
+import { globalStyles, appStyles } from "../src/components/navGlobalStyles";
 
-export { CustomComponents, FyllUtRouter, Template, AppConfigProvider, AmplitudeProvider };
+export { CustomComponents, FyllUtRouter, Template, AppConfigProvider, AmplitudeProvider, globalStyles, appStyles };
```
Comment (it-vegard) [inline lib/index.js:8]: No need to fix now, but I think one import/export called "styles" will be scale better. Should change if we add another grouping later on.

File: src/components/navFormStyle.js [hunk 2]
```diff
@@ -13,6 +25,9 @@ export default {
     gridTemplateColumns: "repeat(auto-fill, minmax(8rem, 1fr))",
     gap: "1rem",
   },
+};
+
+const styles = {
```
Comment (it-vegard) [inline src/components/navFormStyle.js:30]: Should break into smaller parts later.

File: src/index.less [hunk 3]
```diff
@@ -1,5 +1,7 @@
+@import "~nav-frontend-core";
```
Comment (it-vegard) [inline src/index.less:1]: No longer used? -> Maybe change back to .css-file?

File: src/overrideFormioStyles.less [hunk 4]
```diff
@@ -96,22 +88,22 @@ body {
 }
 //Skjemagruppe -- slutt
 
-//Alle skjemafelter -- start
+/*//Alle skjemafelter -- start
```
Comment (it-vegard) [inline src/overrideFormioStyles.less:91]: Should  remove commented out code

**Review by it-vegard — APPROVED** [review]
Approved, but should fix the two last comments (commented out code and unused import/revert to .css file) before merging. The rest can wait until later.

👍 💯 🏆 🧑‍🤝‍🧑 🐐 💨

File: src/index.less (hunk 3 above)
Comment (rrsslnd) [reply src/index.less]: Done

File: src/overrideFormioStyles.less (hunk 4 above)
Comment (rrsslnd) [reply src/overrideFormioStyles.less]: Done

**Pull request merged**